hadoop-common-project/hadoop-common/src/main/java/org/apache/hadoop/ipc/DecayRpcScheduler.java

```diff
@@ -178,6 +188,7 @@
   private static final double PRECISION = 0.0001;
   private MetricsProxy metricsProxy;
   private final CostProvider costProvider;
+  private Set<String> serviceUsernames;
```

Comment:
nit nit: usually it is `UserName` instead of `Username`, so better follow that pattern.

Reply:
Thanks for your review. Updated PR following `UserName`.